stock_no_extra_move/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{module_title}
```

Comment:
Please complete this or follow OCA readme generator conventions https://github.com/OCA/maintainer-tools/pull/339